packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -62,10 +62,10 @@ const SUBJECT = {
 }
 
 export function LightCurveViewerContainer({
+  data = null,
   drawFeedbackBrushes = DEFAULT_HANDLER,
   feedback = false,
   onKeyDown = DEFAULT_HANDLER,
-  subject = SUBJECT,
   loadingState = asyncStates.initialized,
   onError = DEFAULT_HANDLER,
   onReady = DEFAULT_HANDLER
```

Comment:
```suggestion
```

These are unused. Should they be used like other subject viewers?

Reply:
I'm not sure. Only @shaunanoordin really understands the LCV code.

Reply:
Looks safe to get rid of them, to me.